emf/src/main/java/com/powsybl/tutorials/emf/EmfTutorial.java

```diff
@@ -152,12 +157,12 @@ private static void igmPreprocessing(Network mergedNetwork, Map<String, Network>
         networks.forEach((name, network) -> {
```

Comment:
This is weird even if it is working, you must used mergedNetwork.getSubnetworks() and iterate.